daemon/prune.go

```diff
@@ -320,6 +321,7 @@ func (daemon *Daemon) localNetworksPrune(ctx context.Context, pruneFilters filte
 	l := func(nw libnetwork.Network) bool {
 		select {
 		case <-ctx.Done():
+			// when the context is cancelled, return true to stop the networks traverse
```

Comment:
nit; "when" should probably be "if"

would it be enough to just mention "return true to stop network traverse"? or is "context cancelled" a special case?